src/data/set/finite.lean

```diff
@@ -710,21 +707,21 @@ lemma card_range_of_injective [fintype α] {f : α → β} (hf : injective f)
 eq.symm $ fintype.card_congr $ equiv.of_injective f hf
 
 lemma finite.exists_maximal_wrt [partial_order β] (f : α → β) (s : set α) (h : set.finite s) :
-  s.nonempty → ∃a∈s, ∀a'∈s, f a ≤ f a' → f a = f a' :=
+  s.nonempty → ∃ a ∈ s, ∀ a' ∈ s, f a ≤ f a' → f a = f a' :=
 begin
   classical,
   refine h.induction_on _ _,
-  { assume h, exact absurd h empty_not_nonempty },
-  assume a s his _ ih _,
+  { exact λ h, absurd h empty_not_nonempty },
+  intros a s his _ ih _,
```

Comment:
sorry, I am not sure I understand why you want to replace `assume` with `intros` here. Replacing it by a lambda in a term proof makes complete sense to me, but in tactic proofs I prefer `assume` because it feels closer to the way we speak mathematics.

Reply:
It's just the style I've seen all over mathlib; I can't find any right now, but I've seen PRs which replace `assume` with `intros` before; I think in some way it's for uniformity

Reply:
(I figured this was just worth approving as is since on balance it was an improvement, and the place to bikeshed would probably be a PR to the style guide)

Reply:
To be clear, I don't mind merging this at all -- this is the only change which is not clear to me, all the other ones are in term mode and for these I clearly prefer the lambda.